### DISCUSSION TITLE
how can i backup / restore dockerized strapi

### DISCUSSION THREAD

**Discussion by jmcazarez** [strapi/strapi#7836] [Legacy Topics]
Hi, how can i backup / restore dockerized strapi ?

**Comment by derrickmehaffy** [comment]
I'm not sure I understand your question? Backup in what context?

**Reply by jmcazarez** [reply]
I have my application running with docker on my server. But, i need to move to another server. 
I don't want to lose the data. My strapi app is running with mongodb (dockerized).

**Reply by derrickmehaffy** [reply]
You will have to make a data dump from Mongo and restore that on the new server.

Something I would strongly advise against is dockerizing mongo, as you run the risk of losing that database.

**Reply by jmcazarez** [reply]
Ok, thanks.

My mongo data is mapping to folder on my host server.

docker-compose
volumes:
      - ./data/db:/data/db
